### DISCUSSION TITLE
Is it possible to use TorchSharp on MacOSX ARM64?

### DISCUSSION THREAD

**Discussion by dbriard** [xamarin/TorchSharp#1109] [Q&A]
Hi all,
Is it possible to use TorchSharp on MacOSX ARM64, and how? 
I cannot found the corresponding cpu package, only macos x64.

If not with TorchSharp, is there any alternative in C#?

Thanks a lot!

**Comment by NiklasGustafsson** [comment]
There's a pull request that's been pending for half a year. I don't have the ability to test it on MacOS / ARM64, so I've left it to the community members who need / want it to help drive that to completion.

https://github.com/dotnet/TorchSharp/pull/903

**Reply by dbriard** [reply]
@NiklasGustafsson I see, it is unfortunate to me that it has not be merged and available on NuGet as I am not a MacOS user and it will probably be tricky for me to build libtorch and TorchSharp for Amd64, but at least it mean it is possible, and I will try to do that once I get hand on the Mac M1 at work.
